tests/test_visitors/test_ast/test_keywords/test_base_exception.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from wemake_python_styleguide.violations.best_practices import (
+    BaseExceptionViolation,
+)
+from wemake_python_styleguide.visitors.ast.keywords import (
+    WrongExceptionTypeVisitor,
+)
+
+use_baseexception = """
+try:
+    execute()
+except BaseException:
+    raise
+"""
+
+
+@pytest.mark.parametrize('code', [
+    use_baseexception,
+])
+def test_use_base_exception(
+    assert_errors, parse_ast_tree, code, default_options,
+):
+    """Testing that `except BaseException:` is restricted."""
+    tree = parse_ast_tree(code)
+
+    visitor = WrongExceptionTypeVisitor(default_options, tree=tree)
+    visitor.run()
+
+    assert_errors(visitor, [BaseExceptionViolation])
+
+
+def test_exception(
+    assert_errors, parse_ast_tree, default_options,
+):
+    """Testing that `except Exception:` is allowed."""
+    tree = parse_ast_tree("""
```

Comment:
Can you please move this example at the same level as `use_baseexception`?

Reply:
Done! I can see that there are many tests where the code snippet is defined in the body of functions such as https://github.com/wemake-services/wemake-python-styleguide/blob/master/tests/test_visitors/test_ast/test_keywords/test_raise_notimplemented.py#L55. 

I will tidy the tests for this particular refactoring in a separate piece of work after this PR to keep this one as a separate block.